src/handwritten/task.py

```diff
@@ -402,6 +402,83 @@ def is_task_done(self):
 
         return is_task_done
 
+
+    def perform_strain_shunt_cal(
+            self, channel="", shunt_resistor_value=10000,
+            shunt_resistor_location=0, shunt_resistor_select=0,
+            shunt_resistor_source=0, skip_unsupported_channels=False):
+        """
+        Performs shunt calibration for the specified channels of the task. 
+        The instances of this polymorphic VI correspond to the type of bridge 
+        sensor. Refer to the calibration procedure for your module for detailed
+        calibration instructions.
+
+        Performs shunt calibration for the specified channels using a strain
+        gage sensor.
+
+        Args:
+            task/channel in: specifies name of the task or a list of virtual 
+                channels to which the operation applies. If you provide a
+                list of virtual channels, NI-DAQmx creates a task automatically. 
+            channels: specifies a subset of virtual channels in the task that you 
```

Comment:
Name mismatch: channel vs. channels, shunt_resistor_value vs. shunt resistor value, etc.

Running Sphinx should catch these. Currently the PR workflow runs Sphinx from Tox, as part of `run_system_tests`, which didn't complete due to issues with the self-hosted GitHub runner.

Please run `poetry run tox -e docs` to test this.

Reply:
Hmm, it didn't warn: https://github.com/ni/nidaqmx-python/actions/runs/7444178717/job/20285452399?pr=469#step:4:3942

I wonder why. Perhaps sphinx-autodoc doesn't validate parameters, or perhaps the slashes and spaces are causing sphinx-napoleon to ignore these parameters.

Anyway, in addition to fixing the docstrings, please build the docs and look at them in a web browser to make sure they are formatted right.